src/components/SidePanels/DetailsPanel/DetailsPanel.tsx

```diff
@@ -40,16 +41,25 @@ const Content: FC<any> = observer(({
 
 const GeneralPanel: FC<any> = inject("store")(observer(({ store, currentEntity }) => {
   const { relationStore } = currentEntity;
+  const historyEnabled = store.hasInterface("annotations:history");
+  const draftsEnabled = isFF(FF_DEV_2290);
+  const hideHistoryTitle = !historyEnabled && draftsEnabled;
 
   return (
     <>
       <Elem name="section">
-        <Elem name="section-head">
-          Annotation History
-          <span>#{currentEntity.pk ?? currentEntity.id}</span>
-        </Elem>
+        {hideHistoryTitle === false && (
```

Comment:
Is there a need for the explicit false check, or could this be shortened to `!hideHistoryTitle` ?

Reply:
not really, just used to avoid `!var` things. can't even remember why. lm fix that